scripts/image_create_fedora_atomic_dl.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash --login
+ATOMIC_URL="https://download.fedoraproject.org/pub/alt/atomic/stable"
+REL_VERSION=$(curl -Ls $ATOMIC_URL |grep -Eo 'Fedora-Atomic-25-[0-9\.]*'|sort -r|head -1)
+DOWNLOAD_URL="$ATOMIC_URL/$REL_VERSION/CloudImages/x86_64/images/$REL_VERSION.x86_64.qcow2"
+IMAGE_NAME="Fedora-Atomic-25"
+OS_DISTRO_PROPERTY="fedora-atomic"
+
+source $(dirname $0)/image_dl_test_deploy.sh "$DOWNLOAD_URL"
```

Comment:
test_deploy? The name ins not fitting if this will be used in production.

Reply:
Context:

$ grep function\ image_ image_functions.sh 
function image_create() {
function image_download() {
function image_test() {
function image_deploy() {

Reply:
...where image_test() has carried that name since 2015-04-01. If renaming/refactoring/sanitization is required, I'd suggest that would be handled via another task.

Reply:
aaah, okay now it makes sense.